addons/website_sale/tests/test_address.py

```diff
@@ -660,6 +660,53 @@ def test_12_recompute_taxes_on_address_change(self):
             "Tax should no longer change after order confirmation",
         )
 
+    def test_two_types_address_updates(self):
+        self.env['ir.config_parameter'].sudo().set_int('auth_password_policy.minlength', 4)
+        user = self.env['res.users'].create({
+            'name': 'test',
+            'login': 'test',
+            'password': 'test',
+        })
+        user_partner = user.partner_id
+
+        invoicing, bad_invoicing = self.env['res.partner'].create([
```

Comment:
Try to make the variables reflect the purpose of the record.
So here its not just the invoicing address
```suggestion
        user_address = self.env['res.partner'].create([
```